tex/texlive-bin/Portfile

```diff
@@ -289,6 +303,145 @@ post-destroot   {
     }
     ln -s ${texlive_mactex_texdistdir}-${version} \
           ${destroot}${texlive_mactex_texdistdir}
+
+    # Install the TeXDist directory structure describing MacPorts' TeX Live.
+    #
+    # It is used to switch between TeX distribution on macOS by tools such as
+    # the TeX Live Utility, the texdist script, the TeX Distribution
+    # Preference Pane, and the LocalTeX Preference Pane.
+    # For more details, see
+    #   - the file TeXDist-description.rtf (installed by MacTeX/BasicTeX in
+    #     /Library/TeX/Distributions),
+    #   - MacTeX/BasicTeX's postinstall script,
+    #   - the postinstall script in https://github.com/TeXLive-M/TeXDist, and
+    #   - http://www.tug.org/mactex/multipletexdistributions.html .
+    #
+    # With respect to TeXDist-description.rtf and the postinstall scripts,
+    # we do not install
+    #   - /usr/texbin,
+    #   - /Library/TeX/Distributions/.DefaultTeX,
+    #   - anything in /Library/TeX outside /Library/TeX/Distributions.
+    # In fact they are useful only when other distributions are installed,
+    # and they install those missing pieces.
+    #
+    # MacTeX/BasicTeX's postinstall (but not the other one) install also
+    # symlinks Local/TEXMF{HOME,VAR,CONFIG} inside Contents, but they are not
+    # documented in TeXDist-description.rtf: we do not install them.
+    #
+    # The directories MacPorts-TeXLive.texdist and MacPorts-teTeX.texdist in
+    # /Library/TeX/Distributions/ and MacPorts-TeXLive and MacPorts-teTeX in
+    # /Library/TeX/Distributions/.FactoryDefaults/ are considered by
+    # TeX Live / MacTeX developers as owned by MacPorts.
+    #
+    # From a private email exchange with Richard Koch (April 2015):
```

Comment:
I would remove this long chunk of private communication.
You can copy-paste it to this PR, but there's probably no need to keep it in the `Portfile`.